tests/conftest.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from API import create_app
+
+class User():
+
+    def __init__(self, firstname, lastname, email, password):
+        self.firstname = firstname
+        self.lastname = lastname
+        self.email = email
+        self.password = password
+
+class Request():
+
+    def __init__(self, date, title, location, priority, description):
+        self.date = date
+        self.title = title
+        self.location = location
+        self.priority = priority
+        self.description = description
+
```

Comment:
Why do you have the User and Request object under tests?

Reply:
I created dummy models for my test_models.py but will delete them when i create the model